Dockerfile

```diff
@@ -28,7 +28,7 @@ FROM toolchain AS tools
 ENV GO111MODULE on
 ENV CGO_ENABLED 0
 ENV GOPATH /go
-RUN curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | bash -s -- -b /bin v1.42.1
+RUN curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | bash -s -- -b /bin v1.42.1
```

Comment:
this is auto generated, you'd run `make rekres` to get the updated file

Reply:
yes, that should be fine

Reply:
I have updated the main branch with the changes, you can now rebase this on top of that

Reply:
Thanks.  I have rebased.